packages/backend/server/src/data/migrations/1708321519830-refresh-unlimited-workspace-feature.ts

```diff
@@ -0,0 +1,16 @@
+import { PrismaClient } from '@prisma/client';
+
+import { Features } from '../../core/features';
+import { upsertFeature } from './utils/user-features';
+
+export class RefreshUnlimitedWorkspaceFeature1708321519830 {
+  // do the migration
+  static async up(db: PrismaClient) {
+    // add unlimited workspace feature
+    const feature = Features[3];
```

Comment:
use `Features.find()` instead? I'm concern that the order of features might be somehow changed but no one gets aware of this code